### PR TITLE
+httpaf.0.6.4

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/httpaf-async/httpaf-async.0.6.4/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune"
```
Comment (mseri) [inline packages/httpaf-async/httpaf-async.0.6.4/opam]:
```suggestion
  "dune" { >= "1.5" } 
```
According to the dune-project file you need a lower bound

File: packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.4/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "faraday-lwt-unix"
+  "httpaf" {>= "0.6.0"}
+  "dune"
```
Comment (mseri) [inline packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.4/opam]:
```suggestion
  "dune" { >= "1.5" }
```

File: packages/httpaf/httpaf.0.6.4/opam [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune"
```
Comment (mseri) [inline packages/httpaf/httpaf.0.6.4/opam]:
```suggestion
  "dune" { >= "1.5" }
```

**Comment by mseri** [comment]
@seliopou if you commit these three changes we could merge 0.6.4 before merging he new 0.6.5

**Comment by kit-ty-kate** [comment]
Let's close this one since 0.6.5 has been merged

**Pull request closed without merging**